src/codecs/music_wav.c

```diff
@@ -1072,7 +1076,7 @@ static SDL_bool LoadAIFFMusic(WAV_Music *wave)
             /* Unknown/unsupported chunk: we just skip over */
             break;
         }
-    } while (next_chunk < file_length && SDL_RWseek(src, next_chunk, RW_SEEK_SET) != -1);
+    } while (next_chunk < file_length && SDL_RWseek(src, next_chunk, RW_SEEK_SET) >= 0);
```

Comment:
this one can simply be `== 0`

Reply:
? I don't think that would be correct

Reply:
Oh hell, my mistake, SDL_RWseek obviously follows lseek, not fseek.